src/libraries/System.Runtime.InteropServices.RuntimeInformation/src/System/Runtime/InteropServices/RuntimeInformation/OSPlatform.cs

```diff
@@ -15,7 +17,10 @@ namespace System.Runtime.InteropServices
 
         public static OSPlatform Linux { get; } = new OSPlatform("LINUX");
 
-        public static OSPlatform OSX { get; } = new OSPlatform("OSX");
+        public static OSPlatform macOS { get; } = new OSPlatform("MACOS");
+
+        [EditorBrowsable(EditorBrowsableState.Never)] // https://github.com/dotnet/runtime/issues/33331#issuecomment-650326500
```

Comment:
You need to add it to ref/ version it does matter here much

Reply:
yeah. you can use the `dotnet msbuild /t:GenerateReferenceAssemblySource` in the src/ directory to automate that (it will also make sure the API is added in the correct order).